man/resolvectl.xml

```diff
@@ -339,13 +339,7 @@
         to be dumped and unambiguously parsed.</para></listitem>
       </varlistentry>
 
-      <varlistentry>
-        <term><option>--legend=</option><replaceable>BOOL</replaceable></term>
-
-        <listitem><para>Takes a boolean parameter. If true (the default), column headers and meta information about the
-        query response are shown. Otherwise, this output is suppressed.</para></listitem>
-      </varlistentry>
-
+      <xi:include href="standard-options.xml" xpointer="legend" />
```

Comment:
`resolvectl` does not have `--quiet`...

Reply:
I dropped that commit. It made sense when `--legend` was allowed, but the text can just stay as i tis.